src/App.tsx

```diff
@@ -1,61 +1,35 @@
 import './App.scss';
+import React, { useState } from 'react';
+import { TodoList } from './components/TodoList';
+import { TodoForm } from './components/TodoForm/TodoForm';
+import { Todo } from './Types/Todo';
+import todosFromServer from './api/todos';
+import { getUserById } from './services/user';
+import { getNewTodoId } from './services/id';
+
+export const initialTodos: Todo[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUserById(todo.userId),
+}));
+
+export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>(initialTodos);
+
+  const addTodo = (todo: Todo) => {
+    const todoWithId = {
+      ...todo,
+      id: getNewTodoId(todos),
```

Comment:
Ensure that the `getNewTodoId` function is not directly used in the `TodoForm` component. It should be managed here in the `App` component and passed as a prop if necessary.